src/drive_info.cpp

```diff
@@ -243,7 +243,7 @@ drive_info get_drive_info(std::string const& path)
 		if (dev.handle() == INVALID_HANDLE_VALUE)
 			continue;
 
-#if _WIN32_WINNT >= 0x601
```

Comment:
I don't understand this change. Is there anything in this block that requires windows 10?